components/create_comment/create_comment.jsx

```diff
@@ -233,6 +233,10 @@ export default class CreateComment extends React.PureComponent {
             this.scrollToBottom();
         }
 
+        if (this.state.draft.message !== '') {
```

Comment:
I'm not sure if this can have side effects. componentDidUpdate is execute every time the props or the state changes, and them, independently of what have changed, if you have a draft in the state, you will go to the bottom. I think this must be done only during the "draft" modification (normally when is loaded for first time).

Maybe something like:
```javascript
if (prevState.draft.message !== this.state.draft.message && this.state.draft.message !== '') {
    this.scrollToBottom();
}

Reply:
I agree on the idea and you're right that this specific change is intended on first load only.   However, I tried like what you've suggested (and other workaround) but it didn't work since that state.draft is being set on componentWillMount and so they are equal on initial load and will not trigger scroll to bottom.
I think it's fine to have it as is as I don't find usual change on props/state other than the usual text/file upload change.

Reply:
`this.handleChange` already call `scrollToBottom` so it would be called twice. How about to use `this.props.rhsExpanded` like `focusTextboxIfNecessary` does?

```diff
- if (this.state.draft.message !== '') {
+ if (!prevProps.rhsExpanded) {
```

Reply:
`rhsExpanded` can't be used or maybe you mean RHS open state (if we'll pass `getIsRhsOpen(state)` as props) which didn't work as well.
What I did is introduced `this.doInitialScrollToBottom` which is used to track whether it needs to scroll to bottom on the first component update.
Please take a look again, thanks

Reply:
new change looks good to me.